examples/fewshot_protonet/README.md

```diff
@@ -0,0 +1,102 @@
+# Prototypical Networks for Few-shot Learning
+
+### 1. Description
+
+This demo implements [Prototypical Networks for Few-shot Learning](https://github.com/jakesnell/prototypical-networks) within the `PyKale` framework. The following instructions are about how to use it.
+
+ProtoNet is for few-shot learning under $N$-Way $K$-Shot settings:
+
+$N$-way: It means the number of classes or categories in one epoch in evaluation. For instance, in a 5-way scenario, data from five classes is fed into the model in an epoch.
```

Comment:
Why only in evaluation? not in training? Why do we need to talk about an epoch here? How does the definition of $N$-way involve epoch? What if I do not train or evaluate in epochs?

BTW, in evaluation, why do we need epochs? Very confusing. Please refer to some authoritative sources and pay attention to your English.

Reply:
Will explain it face to face.